router/h2/src/e2e/scala/io/buoyant/router/h2/ConcurrentStreamsEndToEndTest.scala

```diff
@@ -12,7 +12,7 @@ class ConcurrentStreamsEndToEndTest
   with ClientServerHelpers {
 
   // Tunable parameters:
-  val concurrencies = Seq(2, 4, 8, 16, 32, 64, 256)
+  val concurrencies = Seq(8)
```

Comment:
to speed up e2e tests (i.e. in ci).  we can always add more, but it felt like overkill for CI.